package.json

```diff
@@ -15,10 +15,8 @@
   "description": "Connect allthethings",
   "devDependencies": {
     "autoprefixer": "^6.3.3",
-    "babel": "5.8.12",
-    "babel-core": "5.8.12",
-    "babel-eslint": "^5.0.0",
-    "babel-loader": "5.3.2",
+    "babel-eslint": "^6.0.3",
+    "babel-plugin-add-module-exports": "^0.1.3",
```

Comment:
Assuming you're using ES6 import syntax consistently through the project, I don't think you need this plugin.